scripts/coderabbit_retry/coderabbit_retry.py

```
@@ -0,0 +1,384 @@
+            timeout=SUBPROCESS_TIMEOUT_SECONDS,
+        )
+    except (OSError, subprocess.TimeoutExpired) as error:
+        LOGGER.error(f"Failed to list PRs for {repo}: {error}")
+        return None
+    if result.returncode != 0:
+        LOGGER.error(f"Failed to list PRs for {repo}: {result.stderr.strip()}")
+        return None
+    try:
+        return json.loads(result.stdout)
+    except json.JSONDecodeError as error:
+        LOGGER.error(f"Failed to parse PR list JSON for {repo}: {error}")
```

Ruff (0.16.1)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)